Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -107,6 +103,9 @@ - (void)establishClient API_AVAILABLE(macos(10.15)) {
           // Ignore unmodified files
           if (!m->event.close.modified) return;
 
+          // Remove from decision cache in case this is invalidating a cached binary.
+          [self removeCacheEntryForVnodeID:[self vnodeIDForFile:m->event.close.target]];
```

Comment:
How is this callable from SNTEndpointSecurityManager.mm?

Reply:
It's implemented further down (and is a no-op) and then overridden in the caching subclass

Reply:
Oops, I see - it's not new.